### DISCUSSION TITLE
I doubt the code has some malicious code which stolen my credit card. Could someone help me?

### DISCUSSION THREAD

**Discussion by washburn99999** [shadowsocks/shadowsocks-windows#3414] [Show and tell]
I only downloaded the windows C# version. But the next several days my credit card was paid from unknown websites. Could someone help me verify that?

**Comment by chenshaoju** [comment]
IMHO, This issue may not be related to Shadowsocks-windows. May I ask if you downloaded Shadowsocks from which site?

**Comment by washburn99999** [comment]
Hi mate,

Thanks for your reply
I downloaded from github.

**Comment by chenshaoju** [comment]
I think it is safe if you download it from here: https://github.com/shadowsocks/shadowsocks-windows/releases .

The higher probability is due to other reasons. 🤔

**Reply by washburn99999** [reply]
Thanks mate for your updates, but I did download from the link which is forwarded by the official website(https://shadowsocks.org/doc/getting-started.html).
After installing, my credit card was stolen, really frustrated!

**Reply by chenshaoju** [reply]
shadowsocks.org is part of Shadowsocks ( https://github.com/shadowsocks/shadowsocks-org/ ), but it is still safe.

All source is open source, I'm using shadowsocks-windows too.
